src/views/AddTenant/addTenant.js

```diff
@@ -43,56 +42,56 @@ export const AddTenant = () => {
 
   useEffect(() => {
     getProperties();
-  }, []);
+  }, [getProperties]);
 
   useEffect(() => {
     axios.post("/api/users/role", {
       userrole: RoleEnum.STAFF,
       name: staffSearchText
     })
-      .then( staffResponse => {
+      .then(staffResponse => {
         let users = staffResponse.data.users;
         let choices = users
-          ? users.map( u => {
-            return { key: u.id, description: `${u.firstName} ${u.lastName}`}
+          ? users.map(u => {
+            return { key: u.id, description: `${u.firstName} ${u.lastName}` };
           })
           : [];
         setStaffSearchResults(choices);
       })
-      .catch( error => {
+      .catch(error => {
         Toast(error.message, "error");
-      })
+      });
   }, [staffSearchText]);
 
   useEffect(() => {
-    let choices = propertyOptions.filter( 
-      p => p.description.includes(propertySearchText))
+    let choices = propertyOptions.filter(
+      p => p.description.includes(propertySearchText));
     setPropertySearchResults(choices);
-  }, [propertySearchText])
+  }, [propertyOptions, propertySearchText]);
 
-  const getProperties = () => {
+  const getProperties = useCallback(() => {
```

Comment:
useCallback is not defined anywhere so it either needs to get imported or created. Not sure what it is 🤔 
Is it actually needed or can we take it out?